src/projects.h

```diff
@@ -226,12 +226,18 @@ typedef struct ARG_list {
 	/* base projection data structure */
 
 
+#ifdef PJ_LIB__
+    /* we need this forward declaration in order to be able to add a
+       pointer to struct opaque to the typedef struct PJconsts below */
+    struct opaque;
```

Comment:
Actually you are right Even, and I will correct this at the next commit to the branch: As I'm trying to eliminate the polymorphic aspect of the PJ-object, the PJ_LIB__ guard you refer to will, as the branch matures, be eliminated, and PJ will be uniform across the entire library.

Thanks for noting this!